client/dist/index.html

```diff
@@ -3,9 +3,20 @@
 
 <head>
   <title>Template</title>
+  <link rel="stylesheet" href="style.css">
 </head>
 
 <body>
+  <!-- Static Skeleton -->
+  <div class='container'>
+    <div class='top'>
+      <div id='musicPlayer'>music player</div>
+    </div>
+    <div class='bottom'>
+      <div id='comments'>comments</div>
+      <div id='suggestedTracks'>suggested tracks</div>
+    </div>
+  </div>
   <div id="app"></div>
```

Comment:
I don't think we'll need a div for the React app view. I believe that the example Fred gave us has the components each rendered directly to the index.html page, which seems to be what you're doing here. 

tl;dr: omit this line